src/System.Private.Xml/src/System.Private.Xml.builds

```diff
@@ -3,6 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Private.Xml.csproj" />
```

Comment:
Doesn't the following in System.Private.Xml.csproj suffice for defaulting to Windows when `OSGroup` is not specified?

```
<PropertyGroup>
   <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
</PropertyGroup>
```